packages/input-option/src/InputOption/InputOption.style.ts

```diff
@@ -91,60 +69,178 @@ export const inputOptionWedge = css`
   }
 `;
 
-export const inputOptionActiveStyles: Record<Theme, string> = {
-  [Theme.Light]: css`
-    outline: none;
-    background-color: ${palette.blue.light3};
-    color: ${palette.blue.dark2};
+export const disabledStyles = css`
+  cursor: not-allowed;
 
-    &:before {
-      transform: scaleY(1) translateY(-50%);
-      background-color: ${palette.blue.base};
-    }
-  `,
-  [Theme.Dark]: css`
-    outline: none;
-    background-color: ${palette.blue.dark3};
-    color: ${palette.blue.light3};
+  &:before {
+    content: unset;
+  }
+`;
 
+export const getFormElementStyle = ({
```

Comment:
How much redundancy can we eliminate in these functions with something like:
```
const contextStyles = renderedContext  === 'menu' ? menuThemeStyles : formThemeStyles;
```

If there's a fair bit of overlap, it might even be worth refactoring `menuThemeStyles` & `formThemeStyles` into a single object, using the `renderedContext` as the key. Then we can write:

```
const contextStyles = renderedContextStyles[renderedContext];
```

Reply:
updated